src/guess-game.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { useState } from "react";
+
+export default function GuessGame({ cheatProbability }) {
+    const [message, setMessage] = useState("Guess a number");
+    const newTarget = () => Math.floor(Math.random() * 1000) + 1;
```

Comment:
@ynonp reminded me that the number should be updated after every correct guess, so this variable does **belong** in a state, and the setter should be called after the correct guess.